library/src/main/java/de/agilecoders/wicket/webjars/util/RecentVersionCallable.java

```diff
@@ -47,10 +45,8 @@ public String call() throws Exception {
      * @return recent version string
      */
     private static String collectRecentVersionFor(final String partialPath) {
-        return Holder.recentVersionProvider.findRecentVersionFor(partialPath);
+        AssetsMap assetsMap = AssetsMap.get();
+        return assetsMap.findRecentVersionFor(partialPath);
     }
 
-    static final class Holder {
```

Comment:
What's is the problem with the Holder classes ?
Do you know what purpose they have now ?